plugins/parsers/registry.go

```diff
@@ -232,6 +238,17 @@ func NewParser(config *Config) (Parser, error) {
 			config.GrokCustomPatternFiles,
 			config.GrokTimezone,
 			config.GrokUniqueTimestamp)
+	case "hep":
+		parser, err = newHEPParser(config.MetricName,
+			config.HepMeasurementName,
```

Comment:
I think `MetricName` and `HepMeasurementName` denote the same thing and will conflict in the parser. Please remove `HepMeasurementName` in favor of `MetricName`.